server/tests/backends/mongo_translator/test_mongo_translator_steps.py

```diff
@@ -69,5 +70,11 @@ def test_mongo_translator_pipeline(mongo_database, case_id, case_spec_file_path)
     df = pd.DataFrame(result)
     if '_id' in df:
         df.drop('_id', axis=1, inplace=True)
-    expected_df = pd.read_json(json.dumps(spec['expected']), orient='table')
+    expected_df = pd.read_json(
+        json.dumps(
+            spec[f'expected_{exec_type}' if f'expected_{exec_type}' in spec else 'expected']
```

Comment:
I guess this is to have specific expectations for mongo in fixtures? But I don't see `expected_mongo` property anywhere. Are they created by the `retrieve_case` function?
If yes, have I have miss any mongo specific declaration in the JSON fixtures?

Reply:
I modified this code because I thought I would end up adding a `expected_mongo` property, but in the end I managed to get the same results than with pandas.

Reply:
:ok_hand: I was afraid of missing something, I understand better now how this ended up there :+1: